harness/determined/launch/autodeepspeed.py

```diff
@@ -237,9 +196,36 @@ def main(train_entrypoint: str) -> int:
 
     full_cmd = pid_server_cmd + cmd + pid_client_cmd + log_redirect_cmd + harness_cmd
 
-    exit_code = subprocess.Popen(full_cmd).wait()
+    # Chief worker also needs to run sshd when using pdsh and multi-machine training.
```

Comment:
tiny nit: move this comment under the if statement, since it only sometimes applies on this line, but it always applies after the if statement